### PR TITLE
LC-2046 - Login Journey

### PR DESCRIPTION
**Pull request by neeraj-jain-cbo**

### CODE REVIEW

File: src/main/resources/templates/400.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<page th:replace="~{fragments/user-layout :: page(~{::title},~{::content})}">
+    <title>Bad request</title>
+
+    <content>
+        <div class="container">
+            <div class="grid-row">
+                <div class="column-two-thirds">
+                    <h1 class="heading-xlarge">Sorry, there is a problem with the service</h1>
```
Comment (SavvasCO) [inline src/main/resources/templates/400.html:8]: The message on the other services is "Sorry, there is a problem with this service" (replace "the" with "this")

File: src/main/resources/templates/400.html
Comment (SavvasCO) [inline src/main/resources/templates/400.html:1]: Might be better to try and match the error page with the [error page in lpg-services](https://github.com/Civil-Service-Human-Resources/lpg-services/blob/idt-develop/src/ui/page/error.html#L1120-L1127).

**Review by SavvasCO — COMMENTED** [review]
Some comments.

File: src/main/resources/templates/400.html (hunk 1 above)
Comment (neeraj-jain-cbo) [reply src/main/resources/templates/400.html]: Review comments applied.

File: src/main/resources/templates/400.html
Comment (neeraj-jain-cbo) [reply src/main/resources/templates/400.html:1]: Review comments applied.

**Pull request merged**